lib/t/news.ex

```diff
@@ -155,10 +200,14 @@ defmodule T.News do
   @doc false
   def local_mark_seen(user_id, news_story_id) do
```

Comment:
please explain this change

Reply:
the previous version didn't allow for update of `last_id`:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/16062322/159709204-0cf4b8b7-ef87-4286-b094-d896b3e7548f.png">

<img width="401" alt="image" src="https://user-images.githubusercontent.com/16062322/159709214-566ca2e7-e7ce-4df6-b8b5-654465ca851a.png">